src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -378,6 +389,17 @@ private function renderRowSeparator(int $type = self::SEPARATOR_MID)
             $markup .= $column === $count - 1 ? $rightChar : $midChar;
         }
 
+        if (null !== $title) {
+            $lenT = Helper::strlenWithoutDecoration($formatter = $this->output->getFormatter(), $formattedTitle = sprintf($format = $this->style->getTitleFormat(), $title));
```

Comment:
Let's use better variable names here to ease future maintenance. For example:

```php
$lenT -> $titleTextLength
$lenM -> $titleMarkupLength
```

Reply:
Renamed :)